### ISSUE TITLE
Molecule idempotence test fails on linux-system-roles.postfix : Configure Postfix

### ISSUE THREAD

**Issue by girishpatil1386** [linux-system-roles/postfix#27]
I'm wrapping your role in our's, to install and configure postfix. Molecule idempotence test fails on the Configure Postfix.

CRITICAL Idempotence test failed because of the following tasks:
*  => linux-system-roles.postfix : Configure Postfix
*  => linux-system-roles.postfix : Configure Postfix
*  => linux-system-roles.postfix : Configure Postfix

I believe adding a tags: molecule-idempotence-notest on Postfix Configure task should fix the test

**Comment by richm** [comment]
which version of the postfix role are you running?  Which version of docker are you using?  did you make any modifications to the molecule configuration?  The reason I ask is because the molecule tests fail for me like this:
```
TASK [linux-system-roles.postfix : Enable Postfix] *****************************
changed: [centos-6]
fatal: [centos-8]: FAILED! => {"changed": false, "cmd": "/usr/bin/systemctl", "msg": "Failed to connect to bus: No such file or directory", "rc": 1, "stderr": "Failed to connect to bus: No such file or directory\n", "stderr_lines": ["Failed to connect to bus: No such file or directory"], "stdout": "", "stdout_lines": []}
fatal: [centos-7]: FAILED! => {"changed": false, "cmd": "/usr/bin/systemctl", "msg": "Failed to get D-Bus connection: No such file or directory", "rc": 1, "stderr": "Failed to get D-Bus connection: No such file or directory\n", "stderr_lines": ["Failed to get D-Bus connection: No such file or directory"], "stdout": "", "stdout_lines": []}
```

at any rate, I thought the idempotency issue was fixed by this: https://github.com/linux-system-roles/postfix/commit/14314822b529520ac12964e0d2938c4bb18ab895 ensure role is idempotent
?

**Comment by girishpatil1386** [comment]
$ cat meta/main.yml
dependencies:
  - name: linux-system-roles.postfix
    tags: never

$ podman --version
podman version 3.0.2-dev

$ ansible --version
ansible [core 2.11.1]

**Comment by richm** [comment]
Ok - looks like there was a problem with importing the postfix role into Galaxy - but that is now fixed - version 1.0.0 of the role should be idempotent - https://galaxy.ansible.com/linux-system-roles/postfix
